app/views/sign_up/completions/show.html.erb

```diff
@@ -36,7 +36,16 @@
               ),
             ) %>
       <% else %>
-        <%= attribute_value.to_s %>
+        <div class="display-flex flex-justify">
+          <div><%= attribute_value.to_s %></div>
```

Comment:
A few comments:

- Is the attributeless `<div>` wrapper necessary?
- Rather than apply this as the "else" affecting most attributes, can we make this an "elsif" targeting email specifically, similar to what's done above with `social_security_number`?

Reply:
Not necessary - I have removed it. 
Revised the conditionals there as recommended.